doc/classes/Parallax2D.xml

```diff
@@ -0,0 +1,45 @@
+			Velocity at which the offset scrolls automatically, in pixels per second.
+		</member>
+		<member name="follow_viewport" type="bool" setter="set_follow_viewport" getter="get_follow_viewport" default="true">
+			If [code]true[/code], this [Parallax2D] is offset by the current camera's position. It may be desired to disable it if the [Parallax2D] is in a [CanvasLayer] separate from the current camera.
+		</member>
+		<member name="ignore_camera_scroll" type="bool" setter="set_ignore_camera_scroll" getter="is_ignore_camera_scroll" default="false">
+			If [code]true[/code], [Parallax2D]'s position is not affected by the position of the camera.
+		</member>
+		<member name="limit_begin" type="Vector2" setter="set_limit_begin" getter="get_limit_begin" default="Vector2(0, 0)">
+			Top-left limits for scrolling to begin. If the camera is outside of this limit, the [Parallax2D] stops scrolling. Must be lower than [member limit_end] minus the viewport size to work.
+		</member>
+		<member name="limit_end" type="Vector2" setter="set_limit_end" getter="get_limit_end" default="Vector2(0, 0)">
+			Bottom-right limits for scrolling to end. If the camera is outside of this limit, the [Parallax2D] will stop scrolling. Must be higher than [member limit_begin] and the viewport size combined to work.
+		</member>
+		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
+			The [Parallax2D]'s offset relative to its parent.
+		</member>
+		<member name="repeat_size" type="Vector2" setter="set_repeat_size" getter="get_repeat_size" default="Vector2(0, 0)">
+			The [Parallax2D]'s [Texture2D] repeating size. Useful for creating an infinite scrolling background. If an axis is set to [code]0[/code], the [Texture2D] will not be repeated.
+			If the texture's axis size multiplied by [member repeat_times] is smaller than the viewport axis, it will not repeat infinitely, as the [Parallax2D] only draws the specified number of instances of the texture at any given time and offsets it to create the effect.
+		</member>
+		<member name="repeat_times" type="int" setter="set_repeat_times" getter="get_repeat_times" default="1">
+			Override the amount of times the texture repeats. Useful for when zooming out with a camera. As the number increases, the copies will spread away from the origin texture, where each odd-numbered copy is added in front of the texture, and even number is added behind.
+		</member>
+		<member name="screen_offset" type="Vector2" setter="set_screen_offset" getter="get_screen_offset" default="Vector2(0, 0)">
+			Can be used in conjunction with [member ignore_camera_scroll] to manually scroll.
+		</member>
+		<member name="scroll_scale" type="Vector2" setter="set_scroll_scale" getter="get_scroll_scale" default="Vector2(1, 1)">
+			Multiplies the [Parallax2D]'s motion. If an axis is set to [code]0[/code], it will not scroll.
```

Comment:
Not sure if the behavior with 0 is worth mentioning. Don't be afraid to contest this, though.
```suggestion
			Multiplier to the final [Parallax2D]'s offset. Can be used to simulate distance from the camera.
```

Reply:
Would it be too much to mention that >1 makes things appear faster, 1 is the same speed as the camera, <1 makes it appear slower, and 0 makes it seem to not move at all?

Reply:
No, something like that is great, actually. Although it may (?) be better to say "closer" or "further" from the camera.